src/appliance-onboarding-script/appliance_setup/avs/avsarconboarder/retriever/arcaddon/arc_add_on_retriever.py

```diff
@@ -0,0 +1,32 @@
+import logging
```

Comment:
Is this a new folder (retriever/arcaddon) that we are creating?
Does it make sense for this file to be inside retriever/cloud_data?

Reply:
The existing pattern is we have a separate folder for arc_add_on deleter(deleter/arcaddon) and creater(creator/arcaddon). Following the existing pattern.

Reply:
I am okay with this pattern. Cloud data retriever would also be okay as we are fetching multiple details there as well. Will let @abhi580 take the call here.